src/components/TextWithEllipses/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import _ from 'underscore';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import stylePropTypes from '../../styles/stylePropTypes';
+
+const propTypes = {
+    /** leading text, first text of the group */
+    leadingText: PropTypes.string.isRequired,
+
+    /** trailing text, second text of the group */
+    trailingText: PropTypes.string.isRequired,
+
+    /** styling for leading and trailing text */
+    textStyle: stylePropTypes,
+
+    /** styling for leading text View */
+    leadingTextParentStyle: stylePropTypes,
+
+    /** styling for trailing text View */
+    trailingTextParentStyle: stylePropTypes,
+
+    /** styling for parent View */
+    wrapperStyle: stylePropTypes,
+};
+
+const defaultProps = {
+    textStyle: {},
+    leadingTextParentStyle: {},
+    trailingTextParentStyle: {},
+    wrapperStyle: {},
+};
+
+const TextWithEllipses = (props) => {
+    const wrapperStyles = _.isArray(props.wrapperStyle) ? props.wrapperStyle : [props.wrapperStyle];
+    const leadingTextParentStyles = _.isArray(props.leadingTextParentStyle) ? props.leadingTextParentStyle : [props.leadingTextParentStyle];
+    const trailingTextParentStyles = _.isArray(props.trailingTextParentStyle) ? props.trailingTextParentStyle : [props.trailingTextParentStyle];
```

Comment:
Why don't we create a small util for this and then use to directly in props. 

Something like 
```
style={[...parseStyleAsArray(props.wrapperStyle])}

```

Reply:
I've added a function to `StyleUtils`. Does that work?

Reply:
Yeah.